bin/aursearch

```diff
@@ -30,16 +30,13 @@ split() {
 }
 
 getjson() {
```

Comment:
I'm not sure why but I'm missing some results in this section.
```
[archie@vbox bin]$ time ./aursearch python | wc -l
6926
[archie@vbox bin]$ time aursearch python | wc -l
[#37eae8 0B/0B CN:1 DL:0B]
6972
```
Possibly a newline is missing somewhere; the AUR does not add newlines by itself. That's why `awk '{print}' was used instead of cat; `-w \n` may also be necessary for curl.